src/routes/publish.ts

```diff
@@ -120,62 +118,51 @@ function checkCurrentFileImport(req: AuthedRequest, res: Response): FileImportDT
         req.session.errors = generateViewErrors(undefined, 500, [
             generateError('session', 'errors.session.current_import_missing', {})
         ]);
-        res.redirect(`/${lang}/${req.i18n.t('routes.publish.start', { lng: lang })}/`);
+        res.redirect(`/${lang}/${req.i18n.t('routes.publish.start', { lng: lang })}`);
```

Comment:
They'll be a URL builder coming in a subsequent PR that will handle this a bit nicer. Something like:

```
res.redirect(getUrl('/path/to/page', { lang }));
```